src/utils/streams/list_stream.js

```diff
@@ -0,0 +1,25 @@
+import { Readable } from 'stream';
+
+/**
+ *  Create a Readable stream that provides the items
+ *  from a list as objects to subscribers
+ *
+ *  @param  {Array<any>} items - the list of items to provide
+ *  @return {Readable}
+ */
+export function createListStream(items) {
```

Comment:
Any reason you didn't use a default argument here rather than the `items || []` check on the next line?